packages/near-membrane-shared/src/Number.ts

```diff
@@ -1,5 +1,9 @@
-const NumberCtor = Number;
+export const NumberCtor = Number;
 
-export const { isFinite: NumberIsFinite, isInteger: NumberIsInteger } = NumberCtor;
+export const {
+    isFinite: NumberIsFinite,
+    isInteger: NumberIsInteger,
+    isNaN: NumberIsNaN,
```

Comment:
Same for any shared Array additions

Reply:
You mean on the LWS repo? Does the formatter use shared values from that package instead? Not entirely sure what you mean because this package doesn't use @locker/shared

Reply:
Yes behind the scene they are remapped.

Reply:
Exposed: https://github.com/salesforce-experience-platform-emu/locker/pull/1469/files